components/input/config.css

```diff
@@ -12,6 +12,7 @@
   --input-text-disabled-color: var(--input-text-bottom-border-color);
   --input-text-disabled-text-color: var(--input-text-label-color);
   --input-text-error-color: rgb(222, 50, 38);
+  --input-text-helper-color: color(var(--color-black) a(26%));
```

Comment:
Could we change this to `--color-text` so it's consistent with the changes made in this PR? https://github.com/react-toolbox/react-toolbox/pull/1766

Reply:
Well, it has to be lighter than the regular text. If you wish to replace `--color-black` with `--color-text` - that's ok!